sys/page_revisions.js

```diff
@@ -261,7 +262,8 @@ class PRS {
     storePageDeletion(hyper, req, revision) {
         return this.storeRestrictions(hyper, req, {
             title: req.params.title,
-            rev: req.params.revision || revision.rev,
+            // We're storing page_deletions with magic 0 rev just to update the static column
```

Comment:
Perhaps also worth mentioning that 0 is chosen so that it will always match for any query for less-or-equal some revision greater than 0.

Reply:
{{done}}